src/wp-includes/script-loader.php

```diff
@@ -3154,6 +3154,7 @@ function wp_enqueue_stored_styles( $options = array() ) {
  *     @type string[]         $deps   Array of registered stylesheet handles this stylesheet depends on.
  *     @type string|bool|null $ver    Stylesheet version number.
  *     @type string           $media  The media for which this stylesheet has been defined.
+ *     @type string|null      $path   Absolute path to the stylesheet, so that it can potentially be inlined.
```

Comment:
Should `'path' => null` be added to the default args below?

Reply:
Not sure if this is needed.